ddio/sdlcontroller.h

```diff
@@ -49,10 +49,10 @@ const int CTID_KEYBOARD = -1, // always -1 for keyboards
 
 const int CTID_EXTCONTROL0 = 0;
 
-class lnxgameController : public gameController {
+class sdlgameController : public gameController {
```

Comment:
Seems there need change most of methods of `sdlgameController` from `virtual void foo()` to `void foo() override`.

Reply:
good call, fixed